ts/Extensions/Boost/WGLRenderer.ts

```diff
@@ -881,10 +889,15 @@ class WGLRenderer {
 
             // The first point before and first after extremes should be
             // rendered (#9962, 19701)
+            // Boost fails to render scatter series with a single point (#21897)
```

Comment:
"Boost fails..." describes a situation that is no longer true, so this comment would be confusing to the future us. This is more precise:
```suggestion
            // Make sure series with a single point are rendered (#21897)
```